components/forms/RadioButton/react.js

```diff
@@ -0,0 +1,35 @@
+import React, { memo } from 'react'
+import PropTypes from 'prop-types'
+import s from './style.module.scss'
+
+const Radio = ({
+  checked, onChange, value, name,
+}) => (
+  <div role="radio" aria-checked={checked} tabIndex={0} className={s.radio}>
+    <input
+      onClick={event => onChange(event.target.value)}
```

Comment:
I like `event => onChange(event.target.value)` a lot, but we should return the full event because that is what people expect (example case, because I was trying to research if there is a preference: https://github.com/JedWatson/react-select/issues/1631)

should just be `onClick={onChange}`, same thing for `onKeyPress`

If you have seen this example in another component, I apologize for changing it.